kratos/input_output/vtk_output.cpp

```diff
@@ -1002,6 +1005,45 @@ void VtkOutput::ForceBigEndian(unsigned char* pBytes) const
 /***********************************************************************************/
 /***********************************************************************************/
 
+VtkOutput::GeometryType::Pointer VtkOutput::ReorderConnectivity(GeometryType::Pointer pGeometry) const
```

Comment:
I think here using a reference for the fct argument would be suitable for avoiding one shared_ptr-copy

Reply:
Okay